library/AndroidManifest.xml

```diff
@@ -3,5 +3,10 @@
           package="com.google.maps.android">
     <uses-sdk android:minSdkVersion="9"/>
     <!-- https://groups.google.com/group/adt-dev/browse_thread/thread/c059c0380998092b/6720093fb40fdaf9 -->
-    <application/>
+    <application>
+        <meta-data
```

Comment:
This was added to allow for unit tests to extend the AndroidTestCase. In particular, we need this to test the changing of icons for markers.